contracts/superclasses/Posts.sol

```diff
@@ -234,14 +240,14 @@ contract Posts is KetlGuarded {
     uint feedId = commentRequest.feedId;
     uint postId = commentRequest.postId;
     uint replyTo = commentRequest.replyTo;
-    // Fetch parent post
-    Post memory parentPost = posts[feedId][postId];
```

Comment:
We use this twice below, without this change the git diff would look better

Reply:
It didn't work for us for some reason